### DISCUSSION TITLE
Re-order cube to address discontiguities

### DISCUSSION THREAD

**Discussion by rebeccaherman1** [SciTools/iris#5542] [Q&A]
Let's say that I extract a region that crosses 360 degrees longitude (for instance, from 300 to 60 degrees). After this, there will be a discontiguity in the bounds, identified by `_discontiguity_in_bounds`, which comes from the fact that the grid is stored from 0 to 360 even though the contiguous region wraps around 360. This discontiguity in turn prevents regrid-ing. 

Is there any function that allows us to change the way the cube is stored so that it sees the bounds as contiguous? 

If so, please help me find it.

If not, there definitely should be.

**Comment by rcomer** [comment]
It sounds like you should do your extraction with `intersection`
https://scitools-iris.readthedocs.io/en/stable/generated/api/iris.cube.html#iris.cube.Cube.intersection

**Comment by rebeccaherman1** [comment]
Thanks @rcomer. I have realized that the phenomenon I was experiencing is actually due to a (seemingly unnecessary) line of code in ESMValCore which takes a second intersection with longitude=(0,360)
